helm/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kiam
-version: 5.5.0
+version: 5.6.1
```

Comment:
Please bump the minor version, rather than patch, as we are making a backwards compatible functionality change

Reply:
@rhysemmas sorry, i don't understand, currently the PR version is 5.6.1, what version you want that i configure?

Reply:
Hey @caiohasouza, Helm charts use semantic versioning to indicate major/minor/patch updates to the functionality of the chart (see https://semver.org/ ). So in this case your PR will introduce a backwards-compatible change, which by semver standards means we should increase the minor version of the chart. The chart version is currently `5.6.1`, so in your PR the chart version should be bumped to `5.7.0`

Reply:
Hi @rhysemmas , thank you for you explanation. Ok, version changed!